drivers/SmartThings/deepsmart/src/client/httpclient.lua

```diff
@@ -0,0 +1,251 @@
+local log = require('log')
+local cosock = require "cosock"
+local http = cosock.asyncify "socket.http"
+local ltn12 = require('ltn12')
+local RestClient = require "lunchbox.rest"
+local utils = require "utils.utils"
+local json = require('dkjson')
```

Comment:
You should use `st.json` instead of `dkjson` for this functionality in drivers

Reply:
We are already modifying the test